pinot-common/src/main/java/org/apache/pinot/common/utils/tls/RenewableTlsUtils.java

```diff
@@ -216,6 +218,20 @@ static void enableAutoRenewalFromFileStoreForSSLFactory(SSLFactory sslFactory, S
           throw new RuntimeException(e);
         }
       });
+
+      // Reloading SSL Factory blindly every day in a new scheduled thread
+      // to prevent any discrepancies with the FileWatcher
+
+      Executors.newSingleThreadScheduledExecutor().scheduleAtFixedRate(() -> {
+        try {
+          reloadSslFactory(sslFactory,
+              keyStoreType, keyStorePath, keyStorePassword,
+              trustStoreType, trustStorePath, trustStorePassword,
+              sslContextProtocol, secureRandom, insecureModeSupplier);
+        } catch (Exception e) {
+          throw new RuntimeException(e);
+        }
+      }, 1200, CERT_RELOAD_JOB_INTERVAL_IN_HOURS, TimeUnit.HOURS);
```

Comment:
1200 is typo. We can use CERT_RELOAD_JOB_INTERVAL_IN_HOURS for initial delay as well.

Reply:
Initial Delay should be in seconds.  I have given 1200 to give an initial delay of 20 minutes so that the FileWatcher can do its job in case there is any change during startup. I will move this to a variable. but 1200 makes sense?

Reply:
Check https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ScheduledExecutorService.html#scheduleAtFixedRate-java.lang.Runnable-long-long-java.util.concurrent.TimeUnit-

unit - the time unit of the initialDelay and period parameters

As written, initialDelay is 1200 hours!

Reply:
Done